examples/pxScene2d/external/buildWindows.bat

```diff
@@ -78,12 +78,12 @@ cd ..
 
 REM --------- LIBNODE
 
-git apply node-v8.15.1_mods.patch
-cd libnode-v8.15.1
+git apply node-v10.15.3_mods.patch
+cd libnode-v10.15.3
 if %buildExternal% == 1 (
-  CALL vcbuild.bat x86 nosign no-optimization static
+  CALL vcbuild.bat x86 nosign no-optimization static openssl-no-asm
```

Comment:
no-optimization doesn't exist in original node. see #1841. you can drop this code for node 10